docs/alerts/alert-rules.md

```diff
@@ -0,0 +1,163 @@
+title: Alert Rules
+description: Alert rules are essential for monitoring your applications and infrastructure. They allow you to set specific conditions under which you will be notified of changes in metrics or events
+
+Alert rules are critical for maintaining the health and performance of your applications and infrastructure. By setting specific conditions, you can get notified on unexpected changes, helping you respond quickly to issues like outages or performance drops.
+
+## Viewing Alert Rules
+
+The **Alert Rules** page ([US](https://apps.sematext.com/ui/alerts/rules) or [EU](https://apps.eu.sematext.com/ui/alerts/rules)) provides an organized list of all defined alert rules within your Apps and offers a clear overview of their current status and configuration. 
+
+![Alert Rules](../images/alerts/alert-rules.png)
+
+Fields available:
+
+- **Priority:** Specifies the importance of the alert rule.
+- **Integrations:** Lists the [notification methods](https://sematext.com/docs/alerts/alert-notifications/) (e.g., Email, Slack, Webhook) tied to the alert.
+- **Alert Rule Name:** The name of the alert rule.
+- **App:** The App to which this alert rule is applied.
+- **Last Notification:** Shows the time of the last alert notification. Clicking on this leads to the specific [alert event](https://sematext.com/docs/alerts/alert-events/) details.
```

Comment:
It's probably actually the last time the AR was triggered. One can turn off notifications.